frontend/src/component/project/Project/ProjectFeatureToggles/ActionsCell/ActionsCell.tsx

```diff
@@ -63,6 +67,16 @@ export const ActionsCell: VFC<IActionsCellProps> = ({
     const id = `feature-${featureId}-actions`;
     const menuId = `${id}-menu`;
 
+    const handleCopyToClipboard = () => {
+        copy(featureId);
+        setIsFeatureNameCopied(true);
+
+        setTimeout(() => {
+            handleClose();
+            setIsFeatureNameCopied(false);
+        }, 1000)
+    };
```

Comment:
See comment above. Is there a reason why the setTimeout is set to 1 second here?

Reply:
Since the copying feature appears in a small modal, closing the action modal after 1 second would be better. Having it hanging in for 3 seconds after copying looked like the closing reaction was a little slower. I am open to any suggestions or thoughts.